assets/lokomotive-kubernetes/bootkube/conditional.tf

```diff
@@ -9,7 +9,6 @@ resource "local_file" "calico" {
     network_mtu                     = var.network_mtu
     network_encapsulation           = indent(2, var.network_encapsulation == "vxlan" ? "vxlanMode: Always" : "ipipMode: Always")
     ipip_enabled                    = var.network_encapsulation == "ipip" ? true : false
-    ipip_readiness                  = var.network_encapsulation == "ipip" ? indent(16, "- --bird-ready") : ""
```

Comment:
This is still used in the template it seems?

Reply:
Yes, thanks :-)